migrations/004.do.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE verifications ADD COLUMN IF NOT EXISTS test_type TEXT NOT NULL DEFAULT 'confirmed';
```

Comment:
Can we safely rename the migration files using a date based id for these?

Reply:
Made this change. I wonder if its worth retroactively renaming the existing migrations for consistency - they can all safely be re-run so would just need to clear out the `migrations` table before deployment.